code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -135,11 +137,99 @@
 		if(H.m_intent == "run")
 			if(footstep > 1)
 				footstep = 0
-				playsound(H, "clownstep", 50, 1) // this will get annoying very fast.
+				playsound(H, step_sound, 50, 1) // this will get annoying very fast.
 			else
 				footstep++
 		else
-			playsound(H, "clownstep", 20, 1)
+			playsound(H, step_sound, 20, 1)
+
+/obj/item/clothing/shoes/clown_shoes/advanced
+	name = "advanced clown shoes"
+	desc = "Only granted to the most devout followers of Honkmother."
+	icon_state = "superclown"
+	item_state = "superclown"
+	flags = NOSLIP
+	var/list/sound_list = list(
+		"Clown squeak" = "clownstep",
+		"Bike horn" = 'sound/items/bikehorn.ogg',
+		"Air horn" = 'sound/items/AirHorn.ogg',
+		"Chewing" = 'sound/items/eatfood.ogg',
+		"Polaroid" = "polaroid",
+		"Gunshot" = 'sound/weapons/Gunshot.ogg',
+		"Ion gun" = 'sound/weapons/ion.ogg',
+		"Laser gun" = 'sound/weapons/Laser.ogg',
+		"Punch" = "punch",
+		"Shotgun" = 'sound/weapons/shotgun.ogg',
+		"Taser" = 'sound/weapons/Taser.ogg',
+		"Male scream" = "malescream",
+		"Female scream" = "femalescream",
+		"Sad trombone" = 'sound/misc/sadtrombone.ogg',
+		"Awooga" = 'sound/effects/awooga.ogg',
+		"Bubbles" = 'sound/effects/bubbles.ogg',
+		"EMP pulse" = 'sound/effects/EMPulse.ogg',
+		"Explosion" = "explosion",
+		"Glass" = 'sound/effects/glass_step.ogg',
+		"Mouse squeak" = 'sound/effects/mousesqueek.ogg',
+		"Meteor impact" = 'sound/effects/meteorimpact.ogg',
+		"Supermatter" = 'sound/effects/supermatter.ogg',
+		"Emitter" = 'sound/weapons/emitter.ogg',
+		"Laughter" = 'sound/effects/laughtrack.ogg',
+		"Mecha step" = 'sound/mecha/mechstep.ogg')
+
+/obj/item/clothing/shoes/clown_shoes/advanced/attack_self(mob/user)
+	if(user.mind && user.mind.assigned_role != "Clown")
+		user << "<span class='danger'>These shoes are too powerful for you to handle!</span>"
+		if(prob(25))
+			if(ishuman(user))
+				var/mob/living/carbon/human/H = user
+				H << sound('sound/items/AirHorn.ogg')
+				H << "<font color='red' size='7'>HONK</font>"
+				H.sleeping = 0
+				H.stuttering += 20
+				H.ear_deaf += 30
+				H.Weaken(3) //Copied from honkerblast 5000
+				if(prob(30))
+					H.Stun(10)
+					H.Paralyse(4)
+				else
+					H.Jitter(500)
+		return
+
+	var/user_loc = user.loc
+	var/self_loc = src.loc //To disallow fucking with shoes from distance
+	var/new_sound = input(user,"Select the new step sound!","Advanced clown shoes") in sound_list
+
+	if(user_loc==user.loc && self_loc==src.loc)
+		step_sound = sound_list[new_sound]
+		user << "<span class='sinister'>You set the step sound to \"[new_sound]\"!</span>"
+	else
+		user << "<span class='warning'>You have to stand still!</span>"
+
+/obj/item/device/soundsynth/verb/ChangeSound()

```

Comment:
I can't fix it as I'm using a phone atm, but if somebody were so nice as to make a PR to my repo fixing that typo, I could merge it
If not, well, I'd be back in a week